### PR TITLE
Template integration

### PR DESCRIPTION
**Pull request by carloslahrssen**
This branch has the following. 

It has all the email templates written in handlebar for every email necessary. It also has a config file for setting variables for the template.

### CODE REVIEW

File: src/email_templates/readme.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+However, most of the major email clients such as Gmail, Outlook and Spark among others have amazing support to rendering tables as well as writing css styles within the HTML file. 
+
+For that reason we went forward with creating our email templates with tables. But don’t worry we did all the heavy lifting so that you wouldn’t need to. 
+
+## So what do I do if I want to add my own styles or additional fields? 
+
+Unfortunately, to change styles across emails at the moment you would need to go to every .handlebars file to change and test the html accordingly. 
+
+With that said there is hope, if you want minimal changes such as the background color or the logo image or the hackathon name, there’s a sweet config file where you can set all of the above to get those changes for every template.
```
Comment (andres-valdes) [inline src/email_templates/readme.md:18]: Maybe refer to the path of the config? `src/config/index.js`.

File: src/config/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const config = {
```
Comment (andres-valdes) [inline src/config/index.js:1]: This is beautiful, have some really cool ideas for when it comes to stitching everything together via smoothie!

**Review by andres-valdes — CHANGES_REQUESTED** [review]

File: src/email_templates/readme.md (hunk 1 above)
Comment (carloslahrssen) [reply src/email_templates/readme.md:18]: Yeah you're right! Forgot to push my changes from the readme. Branch is updated.

File: src/config/index.js (hunk 2 above)
Comment (carloslahrssen) [reply src/config/index.js:1]: Can't wait for that sweet smoothie

**Review by andres-valdes — APPROVED** [review]
lgtm/bsn

**Pull request merged**